snippets/Date/Date.IsAfterDate.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+
+namespace conplement.snippets.Date
+{
+    /// <summary>
+    /// Partial class for date snippets
+    /// </summary>
+    public static partial class Date
+    {
+        /// <summary>
+        /// Check if a date is after another date.
+        /// </summary>
+        /// Documentation for DateTime: https://docs.microsoft.com/de-de/dotnet/api/system.datetime
+        public static bool IsAfterDate(this DateTime dateTime1, DateTime dateTime2)
```

Comment:
see https://github.com/GuepardoApps/30-seconds-of-c-sharp/commit/e7f4504d50ab70d510445b4942e975b69c69cf56#r31452178